python/common.mk

```
@@ -128,9 +128,9 @@ $(ARCWRAPPERS): %_wrap.cpp: $(top_srcdir)/swig/%.i $(top_srcdir)/swig/Arc.i $(PY
         done
 
 install-data-hook:
-	if test -n "$(PYTHON_SOABI)" ; then \
+	if test -n "$(PYTHON_EXT_SUFFIX)" ; then \
```

This is not the correct change here. The test is to ensure that the move is actually a move. On Python 2 there should be no move (since PYTHON_EXT_SUFFIX is .so there).
The build fails with:
```
if test -n ".so" ; then \
  mv /builds/nordugrid/arc/rpmbuild/BUILDROOT/nordugrid-arc-20210309123401-1.el7.x86_64/usr/lib64/python2.7/site-packages/_arc.so \
  /builds/nordugrid/arc/rpmbuild/BUILDROOT/nordugrid-arc-20210309123401-1.el7.x86_64/usr/lib64/python2.7/site-packages/_arc.so ; \
fi
mv: '/builds/nordugrid/arc/rpmbuild/BUILDROOT/nordugrid-arc-20210309123401-1.el7.x86_64/usr/lib64/python2.7/site-packages/_arc.so' and '/builds/nordugrid/arc/rpmbuild/BUILDROOT/nordugrid-arc-20210309123401-1.el7.x86_64/usr/lib64/python2.7/site-packages/_arc.so' are the same file
make[6]: *** [install-data-hook] Error 1
```

---

We still build for python 2 for older distributions (e.g. RHEL/CentOS 7). The configure script can build for 2 different python versions, so you can build python bindings for both python 2 and 3:
```
  --with-python=(PYTHON)  specify python program from PATH
  --with-altpython=(PYTHON)
                          specify alternative python program from PATH
```

---

Ah of course, sorry about that. How about:
```suggestion
	if [ "$(PYTHON_EXT_SUFFIX)" != ".so" ]; then \
```

---

@ellert is. the suggested change good? Then we can commit the suggestion.